test/WasmTasksTests/WasmTasksTests.csproj

```diff
@@ -25,9 +27,14 @@
     <Reference Include="Microsoft.Build.UnitTests.Shared">
       <HintPath>$(MSBuildThisFileDirectory)\..\..\msbuild\artifacts\bin\Microsoft.Build.UnitTests.Shared\Debug\net8.0\Microsoft.Build.UnitTests.Shared.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Build.Utilities.Core">
```

Comment:
I don't think I want that. I build the Microsoft.Build.UnitTests.Shared DLL (which is not a NuGet package) from the msbuild git submodule. It seems more logical that I use the output of the same build process for all msbuild DLLs I depend on rather than getting some from NuGet and some from local DLLs. I had an issue of incompatible versions of these dlls and this lets me control the version by specifying a msbuild commit in the submodule.